test/templates/pages/empty.njk

```diff
@@ -0,0 +1,26 @@
+{% extends "./layouts/page.njk" %}
+{% set page_title = "Table" %}
+{% set page_intro = "Page level help message goes here" %}
+{% set is_landing_page = false %}
+{% set show_toolbar = true %}
+
+{% import "../layout/component.njk" as layout %}
+
+{% block page_header -%}
+<header class="tn-action-bar">
+    <h1 class="tn-action-bar__title">
+        Page Title
+    </h1>
+</header>
+{%- endblock %}
+
+{% block page_content %}
+
+{%- call layout.section(modifier="full-bleed") -%}
+        <p class="tn-has-text-align-center tn-has-text-italic">
+            <span class="tn-has-text-bold">There are no items on this page.</span><br>
```

Comment:
Just use `strong` here instead. It's a legitimate use of the element. The italic helper is OK in this instance since the usage is more decorative than semantic.